specification/iotcentral/resource-manager/Microsoft.IoTCentral/stable/2020-03-01/examples/Apps_CheckNameAvailability.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {
+    "subscriptionId": "00000000-0000-0000-0000-000000000000",
+    "api-version": "2018-09-01",
```

Comment:
Hello. Looks like this PR doesn't change the api-version. Does it work in progress?

Reply:
fixed